app/templates/signed-in.html

```diff
@@ -9,6 +9,9 @@
         <h1 class="saturn">Congratulations. You are signed in.</h1>
         <p>You can now access protected endpoints.</p>
         <p>To sign out please <a href="{{ url_for('logout') }}" > select here.</a> </p>
+        {% if data.logged_in == "True" %}
```

Comment:
I removed this clause completely and renamed the template to not-signed-in.html as we only now visit this template if we try to access a page before we have signed in.